ml-agents-envs/mlagents/envs/tests/test_subprocess_env_manager.py

```diff
@@ -28,35 +29,51 @@ def __init__(self, worker_id, resp=None):
 class SubprocessEnvManagerTest(unittest.TestCase):
     def test_environments_are_created(self):
         SubprocessEnvManager.create_worker = MagicMock()
-        env = SubprocessEnvManager(mock_env_factory, 2)
+        env = SubprocessEnvManager(
+            mock_env_factory, EngineConfig(80, 80, 1, 20.0, -1), 2
```

Comment:
How about adding a .default_config() staticmethod to EngineConfig? (yes, you can do that to NamedTuples)